src/newrelic/processor_test.go

```diff
@@ -295,16 +333,134 @@ func TestProcessorHarvestCleanExit(t *testing.T) {
 	// supportability and reporting metrics are added
 	m.clientReturn <- ClientReturn{} /* metrics */
 	m.clientReturn <- ClientReturn{} /* events */
+	m.clientReturn <- ClientReturn{} /* usage metrics */
 
 	m.p.CleanExit()
 
-	<-m.clientParams /* ditch metrics */
-	cp := <-m.clientParams
+	<-m.clientParams        /* ditch metrics */
+	cp := <-m.clientParams  /* custom events */
+	cp2 := <-m.clientParams /* usage metrics */
 
 	expected := `["one",{"reservoir_size":5,"events_seen":1},[half birthday]]`
 	if string(cp.data) != expected {
 		t.Fatalf("expected: %s \ngot: %s", expected, string(cp.data))
 	}
+
+	time := strings.Split(string(cp2.data), ",")[1]
+	usageMetrics := `["one",` + time + `,` + time + `,` +
+		`[[{"name":"Supportability/c/specific_collector.com/Output/Bytes"},[2,1315,0,0,0,0]],` +
+		`[{"name":"Supportability/c/specific_collector.com/custom_event_data/Output/Bytes"},[1,60,0,0,0,0]],` +
```

Comment:
I thought the spec didn't indicate it needed the specific collector.  Thought this was mainly `Collector`, `OTLP`, or `InfiniteTracing`

Here's example output:
```
[
  {
    “name”:”Supportability/DotNET/Collector/Output/Bytes”
  },
  [10, 1000, 550, 0, 0, 0]
]
```

Reply:
Nice catch! I've addressed the collector name and added 8T usage metrics here [feat(daemon): add 8T data usage metrics](https://github.com/newrelic/newrelic-php-agent/pull/507/commits/69b03e1abc731d339dd809952cd1ee149bc27f05)

Reply:
Nice addition and impressive turnaround time :)